azure-proto-compute/Extensions/ResourceGroupExtensions.cs

```diff
@@ -3,58 +3,114 @@
 namespace azure_proto_compute
 {
     /// <summary>
-    /// Naming of these methods
+    /// A class to add extension methods to ResourceGroup.
     /// </summary>
     public static class ResourceGroupExtensions
     {
         #region VirtualMachines
-        public static VirtualMachineOperations VirtualMachine(this ResourceGroupOperations resourceGroup, string name)
+        /// <summary>
+        /// Gets the VirtualMachineOperations for a VirtualMachine.
+        /// </summary>
+        /// <param> The <see  cref="[ResourceGroupOperations]" /> instance the method will execute against. </param>
+        /// <param name="vmName"> The name of the VirtualMachine. </param>
+        /// <returns> Returns a response with the <see cref="VirtualMachineOperations"/> operation for this resource. </returns>
```

Comment:
The "Returns a response" wording is slightly out of context here.  It comes from here https://github.com/m-nash/azure-proto-sdk/blob/master/XmlDocStandard.md#method which is referring to an http response.

Here we are simply returning an instance and we might use the class text again with a small modification
https://github.com/m-nash/azure-proto-sdk/blob/master/XmlDocStandard.md#standard-class-documentation

```suggestion
        /// <returns> Returns an object representing the operations that can be performed over a specific VirtualMachine.</returns>
```